src/test/java/uk/gov/pay/webhooks/ledger/pact/LedgerServiceConsumerTest.java

```diff
@@ -30,19 +31,26 @@ public class LedgerServiceConsumerTest {
 
     @Mock
     WebhooksConfig configuration;
-    
+
     private LedgerService ledgerService;
-    
+
     @Before
     public void setUp() {
         when(configuration.getLedgerBaseUrl()).thenReturn(ledgerRule.getUrl());
-        ledgerService = new LedgerService(HttpClient.newBuilder().connectTimeout(Duration.ofSeconds(5)).build(), configuration);
+
+        ClientBuilder clientBuilder = ClientBuilder.newBuilder();
+        Client client = clientBuilder.build();
+
+        clientBuilder.connectTimeout(5, TimeUnit.SECONDS);
+        client.register(RestClientLoggingFilter.class);
```

Comment:
If we had a `RestClientFactory` type like in our other apps, we could call this to give us a client with the correct configuration rather than duplicating it.

This is what we do in adminusers, for example.